### ISSUE TITLE
Window Action Icons hard to identify for qube with colour black

### ISSUE THREAD

**Issue by rwiesbach** [QubesOS/qubes-issues#7196]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)
If the black colour is used for a qube, it is very hard to identify minimize/maximize/close icons

### The solution you'd like
The parts of the icon, that are painted black with the other colours, should have a light gray or other colour that makes them easier to identify

**Comment by DemiMarie** [comment]
Paging @ninavizz

**Comment by ninavizz** [comment]
Yeah... again, pointing to #6414 as a response to #5800—and ask @andrewdavidwong if this should qualify as a duplicate of #5800, as I see them as the same core problem: Qubes OS is the only OS to uniquely color window frames differently, per different needs, and as such this is just gonna happen. 

All existing Linux window managers are going to want to have one colorway for window frames—because that's how all work. If we want one that works well we Qubes OS, we'll need to make it ourselves. Could easily be an adaptation of an existing XFCE theme, but I'd suggest creating one for any/all window managers we support (so XFCE and KDE and i3 for now, and possibly Gnome in the future).

**Comment by DemiMarie** [comment]
I intend to implement client-side decorations in the GUI daemon in the future.  This means that window decorations are drawn by the GUI daemon, not the X server or Wayland compositor.  In turn, this allows ditching a bunch of window manager-specific code and is a prerequisite for Wayland support.

**Comment by ninavizz** [comment]
@DemiMarie I have no idea how that is relevant to this issue—which I don't say to be rude. I honestly don't. :D 

That said, I do feel the underlying issue is a very real problem for users, that needs a simple and clear near-term solution: which is simply, that there is not one window theme that has zero color/legibility issues with Qubes window frame colorization. Without delving into rabbit-holes and other opportunities, I'd like this to get discussed at some time and prioritized for some kind of 2022 address.

**Comment by ninavizz** [comment]
edit: long-term I see your point as relevant @DemiMarie but in an immediately actionable fashion that can be meaningfully acted upon given the team's capacity, I give it the confused-dog look and defer to @andrewdavidwong 😸

**Comment by DemiMarie** [comment]
@ninavizz it is *not* immediately actionable, due to time constraints :laughing:.

**Comment by andrewdavidwong** [comment]
> Yeah... again, pointing to #6414 as a response to #5800—and ask @andrewdavidwong if this should qualify as a duplicate of #5800, as I see them as the same core problem: Qubes OS is the only OS to uniquely color window frames differently, per different needs, and as such this is just gonna happen.

Yes, this does seem like a duplicate.

-----

This appears to be a [duplicate](https://www.qubes-os.org/doc/issue-tracking/#new-issues-should-not-be-duplicates-of-existing-issues) of an existing issue. If so, please comment on the appropriate existing issue instead. If anyone believes this is not really a duplicate, please leave a comment briefly explaining why. We'll be happy to take another look and, if appropriate, reopen this issue. Thank you.
